tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -58,18 +60,27 @@ def create(self, request, *args, **kwargs):
         """Override create to upload in case of successful scan."""
         response = super().create(request, *args, **kwargs)
 
-        # Upload to ACF-TITAN only if file is passed the virus scan and created
+        # only if file is passed the virus scan and created successfully will we perform side-effects:
+        # * Send to parsing
+        # * Upload to ACF-TITAN
+        # * Send email to user
```

Comment:
love this documentation. hope we can get a little workflow captured in technical documentation, maybe once TANF Section 1 parsing is complete? @andrew-jameson

Reply:
Yes, I think we should hold off on any documentation until we have completed #1354 , #1121 , and #1122 . Things are fluid architecture wise at the moment.